casbin/third_party/Cparse/packToken.h

```diff
@@ -0,0 +1,152 @@
+#ifndef PACKTOKEN_H_
+#define PACKTOKEN_H_
+
+#ifdef CASBIN_EXPORTS
+#define META_CLASS_API __declspec(dllexport)
+#define PMETA_API __declspec(dllexport)
+#define PACKTOKEN_API __declspec(dllexport)
+#define PTYPE_API __declspec(dllexport)
+#else
+#define META_CLASS_API __declspec(dllimport)
+#define PMETA_API __declspec(dllimport)
+#define PACKTOKEN_API __declspec(dllimport)
+#define PTYPE_API __declspec(dllimport)
+#endif
+
+#include <string>
+#include <unordered_map>
+#include"../../rbac/role_manager.h"
+
+enum myTypes {
+    PTYPE = 0x24,
+    PMETA = 0x25
+};
+
+
+class META_CLASS_API MetaClass {
+public:
+    virtual std::unordered_map<string, packToken> GetMap() {
+        return std::unordered_map<string, packToken>();
+    };
+};

```

Comment:
You shouldn't modify a third party library directly. In the future commits must remove your code from this file.